### ISSUE TITLE
Dark theme : black on black in file conflict popup

### ISSUE THREAD

**Issue by matthiasbe** [nextcloud/server#22748]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

Please excuse me if there is a duplicate I couldn't find.

### Steps to reproduce
1. Enable dark theme
2. Upload an already existing file

### Expected behaviour
I can read the text in the confict popup

### Actual behaviour
The text is in a black font, so I can't read it

![image](https://user-images.githubusercontent.com/9254232/92602611-a550f900-f2ae-11ea-962f-dfecfd5fd6c4.png)

Nextcloud version : 19.0.2

**Comment by MorrisJobke** [comment]
cc @nextcloud/designers

**Comment by SR4ven** [comment]
Same problem for file and folder sizes when loading the website for the first time. Fixes itself after the first page reload.
Can be replicated using the private mode of browsers (Firefox in my case).

**Comment by szaimen** [comment]
https://github.com/nextcloud/server/issues/21069?

**Comment by matthiasbe** [comment]
Hi @szaimen Deleting the cache does not solve the problem. I had this problem since NC 17 and it is always present if I remember correctly.

**Comment by szaimen** [comment]
I can reproduce this on NC21.0.2
